login/DataConnection.js

```diff
@@ -0,0 +1,13 @@
+const Sequelize = require("sequelize");
```

Comment:
Sorry, what?

Reply:
What "what"?

Reply:
sorry,i dont undestant what made you indignant